src/convertToObject.js

```diff
@@ -6,7 +6,27 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const cssPropertiesDotComa = sourceString
+    .split(';')
+    .map((item) => item.trim())
+    .filter((item) => item.length > 0);
+
+  const cssPropertiesDoubleDot = cssPropertiesDotComa
+    .map((item) => item.split(':'))
+    .map((splittedCssProperty) =>
+      // eslint-disable-next-line prettier/prettier
+      splittedCssProperty.map((propertyPart) => propertyPart.trim()));
+
+  const cssPropertiesObject = cssPropertiesDoubleDot.reduce(
+    (acc, [key, value]) => {
+      acc[key] = value;
+
+      return acc;
+    },
+    {},
+  );
```

Comment:
[CODE PRACTICE] - The `reduce` method is correctly used here to transform the array into an object. However, consider using object destructuring in the parameter of the reduce callback to enhance readability.